src/main/java/svnserver/server/SessionContext.java

```diff
@@ -33,6 +33,8 @@
  * @author Artem V. Navrotskiy <bozaro@users.noreply.github.com>
  */
 public final class SessionContext {
+  static final ThreadLocal<SessionContext> LOCAL = new ThreadLocal<>();
```

Comment:
Why you need this?

Reply:
Get current socket user and repo info at any where in socket thread.
All lfs read interface not have `User` parameter, and repo info, url path mapper file path